.travis.yml

```diff
@@ -12,6 +12,8 @@ env:
   global:
   - secure: IbrybaUaJDiwNh8VodFfTIAdSw2Nl4DGRh2oDIO2vL8SPgGilwxY0go5DWrxAG0aP+YIXFkK+SpblFppQqqxnAoLVYmd7kFSJoOZraNqmcJqUYSzrHrWArmuiGzY8gMgOWTsnbzVVNRldN8tpoSwpQB/+YhCtbYb9eqvw+u5Izk=
   - secure: grvHX6c0ReQwRsuQZferI+uyGsKlubujURKuJB8coNdA9lgZcvf+nmCkNBpSML83hSjQkJUTt8XnucNNgL/1VxUhgy0O/wkTUEmcDAU1b80As8NJkbreJcc3HFASA68wou4HoJsFmJV2cpBYoAJJSX+HyT6mYd6pBh09tPfOeBg=
+  matrix:
+  - TEST_COMMAND="run test:all:parallel"
```

Comment:
is this matrix part necessary? why not just inline it in the script section?

Reply:
- One thing is, I missed seeing a definition of the command run in the travis UI.
- Second thing is, I updated the test:all:parallel npm script to fix the issue and then totally didn't realize the .travis.yml didn't even use that script, and I might have noticed it if the UI showed the command that was run.